### PR TITLE
[AzureCLI] Support being able to change subscription name

### PR DESCRIPTION
**Pull request by ds-ms**

### CODE REVIEW

File: Tasks/AzureCLIV1/task.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "demands": [],
     "version": {
         "Major": 1,
-        "Minor": 132,
+        "Minor": 136,
```
Comment (sachinma) [inline Tasks/AzureCLIV1/task.json]: Should it be 137 or you plan to port this to 136?

File: Tasks/AzureCLIV1/azureclitask.ts [hunk 2]
```diff
@@ -100,12 +100,12 @@ export class azureclitask {
...
         var servicePrincipalKey: string = tl.getEndpointAuthorizationParameter(connectedService, "serviceprincipalkey", false);
         var tenantId: string = tl.getEndpointAuthorizationParameter(connectedService, "tenantid", false);
-        var subscriptionName: string = tl.getEndpointDataParameter(connectedService, "SubscriptionName", true);
+        var subscriptionID: string = tl.getEndpointDataParameter(connectedService, "SubscriptionID", true);
         //login using svn
         this.throwIfError(tl.execSync("az", "login --service-principal -u \"" + servicePrincipalId + "\" -p \"" + servicePrincipalKey + "\" --tenant \"" + tenantId + "\""));
         this.isLoggedIn = true;
         //set the subscription imported to the current subscription
-        this.throwIfError(tl.execSync("az", "account set --subscription \"" + subscriptionName + "\""));
+        this.throwIfError(tl.execSync("az", "account set --subscription \"" + subscriptionID + "\""));
```
Comment (sachinma) [inline Tasks/AzureCLIV1/azureclitask.ts]: Verify the casing of subscriptionID

Comment (ds-ms) [reply Tasks/AzureCLIV1/azureclitask.ts]: What do you mean by casing? That's a variable.

File: Tasks/AzureCLIV1/task.json (hunk 1 above)
Comment (ds-ms) [reply Tasks/AzureCLIV1/task.json]: No, changed it to 137 👍

**Pull request merged**